docs/reference/sql/functions.md

```diff
@@ -11,5 +15,140 @@ functions in DataFusion. These functions include:
 In summary, GreptimeDB supports all SQL aggregate functions and scalar functions in DataFusion. Users can safely
 use these rich built-in functions in GreptimeDB to manipulate and analyze data.
 
-# GreptimeDB Functions
-TODO @dennis
+### `arrow_cast`
+
+`arrow_cast` function is from DataFusion's [`arrow_cast`](https://arrow.apache.org/datafusion/user-guide/sql/scalar_functions.html#arrow-cast). It's illustrated as:
+
+```sql
+arrow_cast(expression, datatype)
+```
+
+Where the `datatype` can be any valid Arrow data type in this [list](https://arrow.apache.org/datafusion/user-guide/sql/data_types.html). The four timestamp types are:
+- Timestamp(Second, None)
+- Timestamp(Millisecond, None)
+- Timestamp(Microsecond, None)
+- Timestamp(Nanosecond, None)
+
+(Notice that the `None` means the timestamp is timezone naive)
```

Comment:
Our http API output them as 

- Timestamp
- TimestampMillisecond
- TimestampMicrosecond
- TimestampNanosecond

I think we can use these names here avoid explaining `None`

Reply:
No, that's the function input, not an API output.

`arrow_cast` function doesn't support expression like `select * from monitor where ts > arrow_cast(1690252336408, 'TimestampMillisecond');`

The error would be `Error during planning: Unsupported type 'TimestampMillisecond'. Must be a supported arrow type name such as 'Int32' or 'Timestamp(Nanosecond, None)'. Error unrecognized word: TimestampMillisecond`